### PR TITLE
Remove outline module from default modules in Viewer

### LINKED ISSUE #2135: Multi-selection in viewer causes black boxes
### Describe the Bug

As a user I'm embedding the viewer as recommended in the [starter project](https://github.com/bpmn-io/bpmn-js-examples/tree/main/starter). When I select elements no outline is shown, I can still react to selected events; this I expect.

However when I select more than one element a black box appears that hides the diagram:

capture vSxTJn_optimized

### Steps to Reproduce

1. open [starter viewer example](https://cdn.statically.io/gh/bpmn-io/bpmn-js-examples/main/starter/viewer.html)
2. select more than one element using `SHIFT + click`
3. see black box appear

### Expected Behavior

* No black box appears
* (Alternative) Selection is not available for the viewer

### Environment

 - Browser: Chrome 110
 - OS: Linux
 - Library version: v14.0.0

### PR DESCRIPTION
**Pull request by misiekhardcore**
### Proposed Changes

We don't want to include outline module by default in `Viewer` modules to not include any visueal, non core functionalities

Closes #2135

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] **Brief textual description** of the changes present
* [ ] **Visual demo** attached
* [ ] **Steps to try out** present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
Consider whether or not to add a test for this.

**Comment by misiekhardcore** [comment]
I would love to add a test for this. I just don't know whats the proper place for it

**Comment by nikku** [comment]
> I'm not sure what is the right way of going with this? We need to merge the changes to diagram-js first but they would still have to be released so we can bump the version in this repo?

For changes that require upstream fix it is fine to prepare + test locally (linked) if things work. We release our libraries continuously, so the next step is to get things merged in diagram-js, release them, and incorporate them into this PR.

File: test/spec/ViewerSpec.js [hunk 1]
```diff
@@ -81,6 +81,19 @@ describe('Viewer', function() {
   });
 
 
+  it('should not include Outline module by default', function() {
+
+    // given
+    var viewer = new Viewer({ container: container });
+
+    // when
+    var outline = viewer.get('outline',false);
```
Comment (nikku) [inline test/spec/ViewerSpec.js:90]:
```suggestion
    var outline = viewer.get('outline', false);
```

Comment (nikku) [reply test/spec/ViewerSpec.js:90]: Good test, clearly managing the expectations!

**Comment by misiekhardcore** [comment]
to make sure I understood correctly - we keep this PR open, fix `diagram-js` and then update this PR with new `diagram-js` version so it passes CI?

**Comment by nikku** [comment]
> [...] we keep this PR open, fix diagram-js and then update this PR with new diagram-js version so it passes CI?

Exactly.

**Comment by misiekhardcore** [comment]
This is now cleaned up and waits for `diagram-js` version bump to incorporate changes done in https://github.com/bpmn-io/diagram-js/pull/943

**Comment by nikku** [comment]
Incorporated diagram-js [v15.1.0](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#1500).

**Comment by barmac** [comment]
This PR is actually a breaking change. The users were previously able to see the selected element which was crucial to view the technical properties, but now they cannot. Cf. https://github.com/camunda/web-modeler/issues/13893
A proper way to fix https://github.com/bpmn-io/bpmn-js/issues/2135 would be to make sure that multiselection is rendered correctly, and not to remove selection outline alltogether.

**Pull request merged**